testing/__init__.py

```diff
@@ -211,7 +210,7 @@ def bdist(self):
         # type: () -> str
         get_pip(
             interpreter=self._interpreter,
-            resolver=ConfiguredResolver(pip_configuration=PipConfiguration()),
+            resolver=ConfiguredResolver.default(),
```

Comment:
There are several `ConfiguredResolver.default()` changes like this in the tests below. This makes those test more robust to combinations not run in CI; I hit a few when testing out this change.